arangod/Agency/State.cpp

```diff
@@ -136,10 +136,90 @@ bool State::persist(index_t index, term_t term,
   return res.ok();
 }
 
+bool State::persistconf(
+  index_t index, term_t term, arangodb::velocypack::Slice const& entry,
+  std::string const& clientId) const {
+
+  LOG_TOPIC(TRACE, Logger::AGENCY)
+    << "persist configuration index=" << index << " term=" << term
+    << " entry: " << entry.toJson();
+  
+  // The conventional log entry-------------------------------------------------
+  Builder log;
+  { VPackObjectBuilder b(&log);
+    log.add("_key", Value(stringify(index)));
+    log.add("term", Value(term));
+    log.add("request", entry);
+    log.add("clientId", Value(clientId));
+    log.add("timestamp", Value(timestamp())); }
```

Comment:
static strings :P

Reply:
this one will have to wait for a real cleanup